docs/Schema.md

```diff
@@ -8,12 +8,23 @@ The following pathway allows you to make and test schema changes locally via mig
    - Ex. `supabase migration new update_track_add_url` if adding a URL column to the track table.
 4. Add your SQL to the auto-generated script file in `/migrations`
 5. Add or update the `seed.sql` data to see populated values for your change if relevant
-6. With Docker running, run `supabase db reset` to apply your changes locally. This might take some time on the first try, since Supabase is using Docker to pull some stuff from the internet (more concrete explanation coming soon ...)
-7. If applying the db changes goes smoothly, go to http://localhost:54323 to see a local version of the Supabase dashboard, where your sample data will be visible.
-8. When done, run `supabase db stop` to stop the local instance of the DB.
 
-After script is approved/merged:
+6. With Docker running, run `supabase db start`
+   - This will take some time on the first run, because the CLI needs to download the Docker images to your local machine. The CLI includes the entire Supabase toolset, and a few additional images that are useful for local development
+7. Run `supabase db reset` to apply your changes locally. This might also take some time. If there are any syntax errors with your migration script or `seed.sql` file, they'll be caught here.
+8. If applying the db changes goes smoothly, go to http://localhost:54323 to see a local version of the Supabase dashboard, where your sample data will be visible. Feel free to add/update data to test out your new schema and any constraints.
+   - Anything you do in this local database won't impact our shared instance
+9. If you want to test new API functionality against the locally-running database, you can point the server
+   at this database by going to `platnm/backend/internal/storage/postgres/storage.go` and modifying the first line of the ConnectDatabase method to `dbConfig, err := pgxpool.ParseConfig("postgresql://postgres:postgres@127.0.0.1:54322/postgres")`
+   - This is a connection string for the local DB
+   - Make sure to switch this back to what it was when you make your PR!
+10. When done, run `supabase db stop` to stop the local instance of the DB.
```

Comment:
Not sure if this is just me, but this wasn't an available command for me so i had to manually shut down docker to stop it

Reply:
so true i put the wrong command 😭